conf/modules.config

```
@@ -81,8 +81,8 @@ if(params.aligner == "cellranger") {
                 path: "${params.outdir}/${params.aligner}/count",
                 mode: params.publish_dir_mode
             ]
-            ext.args = {"--chemistry ${meta.chemistry} --create-bam true " + (meta.expected_cells ? "--expect-cells ${meta.expected_cells}" : '')}
-            time   = { 240.h  * task.attempt }
+            ext.args = {"--chemistry ${meta.chemistry} --create-bam ${params.save_align_intermeds}" + " " + (meta.expected_cells ? "--expect-cells ${meta.expected_cells}" : '')}
+            time   = { check_max( 240.h  * task.attempt, 'time'    ) }
```

Afaik check_max got removed in recent templates because this can now be handled by nextflow directly. 

```suggestion
            time   = {  240.h  * task.attempt, 'time'     }
```